watertap/unit_models/coag_floc_zo_jartest.py

```diff
@@ -0,0 +1,623 @@
+        #       H. Rugner, M. Schwientek,B. Beckingham, B. Kuch, P. Grathwohl,
+        #       Environ. Earth Sci. 69 (2013) 373-380. DOI: 10.1007/s12665-013-2307-1
+        self.slope = Var(
+            self.flowsheet().config.time,
+            initialize=1.86,
+            bounds=(1e-8, 10),
+            domain=NonNegativeReals,
+            units=pyunits.mg/pyunits.L,
+            doc='Slope relation between TSS (mg/L) and Turbidity (NTU)')
+
+        self.intercept = Var(
+            self.flowsheet().config.time,
+            initialize=0,
+            bounds=(0, 10),
+            domain=NonNegativeReals,
+            units=pyunits.mg/pyunits.L,
+            doc='Intercept relation between TSS (mg/L) and Turbidity (NTU)')
+
+        self.initial_turbidity_ntu = Var(
+            self.flowsheet().config.time,
+            initialize=50,
+            bounds=(0, 10000),
+            domain=NonNegativeReals,
+            units=pyunits.dimensionless,
+            doc='Initial measured Turbidity (NTU) from Jar Test')
+
+        self.final_turbidity_ntu = Var(
+            self.flowsheet().config.time,
+            initialize=1,
+            bounds=(0, 10000),
+            domain=NonNegativeReals,
+            units=pyunits.dimensionless,
+            doc='Final measured Turbidity (NTU) from Jar Test')
+
+        self.chemical_doses = Var(
+            self.flowsheet().config.time,
+            self.config.chemical_additives.keys(),
+            initialize=0,
+            bounds=(0, 100),
+            domain=NonNegativeReals,
+            units=pyunits.mg/pyunits.L,
+            doc='Dosages of the set of chemical additives')
+
+        self.chemical_mw = Param(
+            self.config.chemical_additives.keys(),
+            mutable=True,
+            initialize=mw_adds,
+            domain=NonNegativeReals,
+            units=pyunits.kg/pyunits.mol,
+            doc='Molecular weights of the set of chemical additives')
+
+        self.salt_mw = Param(
+            self.config.chemical_additives.keys(),
+            mutable=True,
+            initialize=mw_salts,
+            domain=NonNegativeReals,
+            units=pyunits.kg/pyunits.mol,
+            doc='Molecular weights of the produced salts from chemical additives')
+
+        self.salt_from_additive_mole_ratio = Param(
+            self.config.chemical_additives.keys(),
+            mutable=True,
+            initialize=molar_rat,
+            domain=NonNegativeReals,
+            units=pyunits.mol/pyunits.mol,
+            doc='Moles of the produced salts from 1 mole of chemical additives')
+
+
+        # Build control volume for feed side
+        self.feed_side = ControlVolume0DBlock(default={
+            "dynamic": False,
+            "has_holdup": False,
+            "property_package": self.config.property_package,
+            "property_package_args": self.config.property_package_args})
+
+        self.feed_side.add_state_blocks(
```

Comment:
`feed_side` was used as the name in RO since the CV is based on the feed-channel of the RO membrane (as opposed to the permeate channel on the other side of the membrane). Functionally, this works fine, but a better name that fits the nature of the unit would be nice for new users trying to understand model structure (i.e., better readability with an appropriate name).

Reply:
Change name 'feed_side' to 'control_volume'